### PR TITLE
Automation sandbox

### PR DESCRIPTION
**Pull request by ouchadam**
Android sample project to show a few screens with different entry points and information!

### CODE REVIEW

File: Sandbox/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        applicationId "novoda.com.sandbox"
```
Comment (ataulm) [inline Sandbox/app/build.gradle]: com.novoda.sandbox

File: Sandbox/app/src/main/java/novoda/com/sandbox/Application.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package novoda.com.sandbox;
+
+public class Application extends android.app.Application {
+
+    private static boolean signedIn = false;
```
Comment (ataulm) [inline Sandbox/app/src/main/java/novoda/com/sandbox/Application.java]: :joy:

File: Sandbox/build.gradle [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
```
Comment (ataulm) [inline Sandbox/build.gradle]: why this?

Comment (ouchadam) [inline Sandbox/build.gradle]: android new project wizard :game_die:

File: Sandbox/app/build.gradle (hunk 1 above)
Comment (ouchadam) [inline Sandbox/app/build.gradle]: new project wizard reverses the package name wattt?!?!?

File: Sandbox/app/src/main/java/novoda/com/sandbox/Application.java (hunk 2 above)
Comment (ouchadam) [inline Sandbox/app/src/main/java/novoda/com/sandbox/Application.java]: account manager needs to watch out, there's a new authentication pattern in town!

**Pull request merged**